store/dataStore.test.ts

```diff
@@ -4,6 +4,12 @@ import { useDataStore } from './dataStore';
 // test values
 const userId = '123';
 const userEmail = 'test@email.com';
+const teamName = "Philadelphia Eagles";
```

Comment:
Now that this is getting more complex, we should create an object that houses these values. Grouping user values and weekly pick values together into their own child objects.

Reply:
Created two separate objects: userData - to hold user's information and gameData - to hold football game's information